plugins/snowflake/dbt/adapters/snowflake/impl.py

```diff
@@ -29,6 +34,14 @@ def _catalog_filter_table(cls, table, manifest):
         )
         return super()._catalog_filter_table(lowered, manifest)
 
+    
+    def get_columns_in_relation(self, relation):
```

Comment:
i don't believe you needed to add this here

Reply:
removed

Reply:
actually need this for the function `has_schema_changed()` in the sf plugin imp.py